### ISSUE TITLE
Produce a regression for evictions

### ISSUE THREAD

**Issue by kevinburke** [ericfischer/housing-inventory#4]
The blog post you wrote has a regression that can predict housing prices pretty well based on three variables: supply, wages, and unemployment.

It would be neat if we could get a similar regression that could predict the number of evictions. (It might just be the same variables but that would also be interesting to know).

**Comment by e-n-f** [comment]
It would be interesting. Do you have the data?

**Comment by kevinburke** [comment]
Oh I assumed we could use the same data that you have, but instead of predicting rent prices, predict evictions. ie. "a 1% increase in housing stock leads to a X% drop in evictions"

**Comment by e-n-f** [comment]
You're right. I forgot that I had already included the count of evictions per year.

Do you see a pattern that predicts it?

**Comment by kevinburke** [comment]
This is the point of running a regression, right? I would guess that increasing the housing supply would decrease evictions, but what do I know...

**Comment by e-n-f** [comment]
You can do the regression, and maybe there are fewer evictions in years with more new construction, but it seems pretty weak (R²=0.0117), so I am not eager to state a relationship between the two.

<img width="635" alt="screen shot 2018-03-03 at 2 08 41 pm" src="https://user-images.githubusercontent.com/1951835/36939812-b1c2af1e-1eec-11e8-977a-21ab9c511010.png">
